static/js/sheetsUtils.js

```diff
@@ -0,0 +1,54 @@
+import Sefaria from "./sefaria/sefaria";
+
+export async function getSegmentObjs(refs) {
+    const segments = [];
+
+    for (const ref of refs) {
+      const text = await Sefaria.getText(ref, { stripItags: 1 });
+      const newSegments = Sefaria.makeSegments(text, false);
+      segments.push(...newSegments);
+    }
+    return segments;
+  }
+
+export async function getNormalEnRef(ref) {
```

Comment:
These two functions can be one function that returns an object `{en: text.ref, he: text.heRef}`. Also, this function is very similar to `Sefaria.getRef()`. The main difference is that `getRef()` returns the full text API result (strangely). You can wrap `getRef()` so that your new function returns `{en: text.ref, he: text.heRef}`. The wrapper function can be called `getNormalRef()` for lack of a better term.